extensions/amp-analytics/0.1/test/test-visibility-manager.js

```diff
@@ -1082,6 +1082,75 @@ describes.realWin('VisibilityManager integrated', {amp: true}, env => {
     });
   });
 
+  it('should wait for readyReportPromise', async() => {
```

Comment:
These tests lives under the VisibilityManager integrated test category, but they don't looks like integrated test to me. 

We can keep these test, but can we add more integrated test that test the resolved state value.

I am imaging two tests
1. with `reportWhen`, and `visiblePercentageMin`.  visible percentage never meet, check the model resolved with correct state. 
2. with `reportWhen`, `visiblePercentageMin`, `continousTimeMin`, continous time never meet, check the model resolved with correct state. 

I feel 2 could probably replace 1. Let me know if you think there's better ways to test the end to end logic here. Thanks.

Reply:
Done. PTAL